README.md

```diff
@@ -1,118 +1,119 @@
 1. Clone this control repository to your laptop/workstation
- - `git clone <repository url>`
- - `cd control-repo`
-
+    * `git clone <repository url>`
+    * `cd control-repo`
 1. Remove this repository as the origin remote
- - `git remote remove origin`
-
+    * `git remote remove origin`
 1. Add your internal repository as the origin remote
- - `git remote add origin <url of your bitbucket repository>`
-
+    * `git remote add origin <url of your bitbucket repository>`
 1. Push the production branch of the repository from your machine up to your git server
- - `git push origin production`
+    * `git push origin production`
 
 ### GitHub
 
 1. Prepare your local git client to authenticate with GitHub.com or a local GitHub Enterprise instance.
-  - https://help.github.com/articles/generating-ssh-keys/
-  - https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/
-
-2. Create a repository called `control-repo` in your user account or organization.  Ensure that "Initialize this repository with a README" is not selected.
-  - https://help.github.com/articles/creating-a-new-repository/
-
-3. Make a note of your repository URL (HTTPS or SSH, depending on your security configuration).
+    * <https://help.github.com/articles/generating-ssh-keys/>
+    * <https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/>
+1. Create a repository called `control-repo` in your user account or organization. Ensure that "Initialize this repository with a README" is not selected.
+    * <https://help.github.com/articles/creating-a-new-repository/>
+1. Make a note of your repository URL (HTTPS or SSH, depending on your security configuration).
+1. Clone this control repository to your laptop/workstation:
+    * `git clone <repository url>`
+    * `cd control-repo`
+1. Remove this repository as the origin remote:
+    * `git remote remove origin`
+1. Add your internal repository as the origin remote:
+    * `git remote add origin <url of your github repository>`
+1. Push the production branch of the repository from your machine up to your git server
+    * `git push origin production`
 
-4. Clone this control repository to your laptop/workstation:
-  - `git clone <repository url>`
-  - `cd control-repo`
+## Code Manager Setup
 
-5. Remove this repository as the origin remote:
-  - `git remote remove origin`
+If you use Puppet Enterprise and have not yet enabled and configured Code Manager, in addition to reading the official [documentation](https://puppet.com/docs/pe/2017.3/code_management/code_mgr.html) for enabling it, you may want to look at the Ramp-Up Program's control repository instead of this one. It's similar to this repo except that it has batteries includes, so to speak. There are pre-built profiles for configuring Code Manager, generating SSH keys, and setting up your Git server to work with Code Manager.
```

Comment:
I would suggest setting this link to /latest/ rather than to /2017.3/. Using latest instead of the version number redirects to the most recent docs version.

Reply:
Thanks @jbondpdx I'll make that change.